lib/ossert/stats/base.rb

```diff
@@ -1,10 +1,23 @@
 # frozen_string_literal: true
 module Ossert
   module Stats
+    # A base class for storing and handling values extracted from the projects.
+    #
+    # All metric-specific methods and variables are creating in runtime via
+    # class instance methods "define_*" (intended to use in subclasses) and
+    # in the constructor.
     class Base
       class << self
         attr_accessor :section, :section_type
 
+        # @!attribute [rw] section
+        # name of the metric section (agility or community)
+        # @return [String] name of the section
+
+        # @!attribute [rw] section_type
```

Comment:
I understood that for section_type here should stay "period" notion. I think it is easier to get than section_type...
So any attribute belongs to some combination of a data section and a period (quarter, year, total).